JASP-Tests/R/tests/testthat/test-bainanovabayesian.R

```diff
@@ -43,13 +43,13 @@ test_that("Descriptives Plot matches", {
 test_that("Descriptive Statistics table results match", {
   table <- results[["results"]][["bainContainer"]][["collection"]][["bainContainer_descriptivesTable"]][["data"]]
   expect_equal_tables(table,
-                      list(60, 48.2744318767126, 49.8666666666667, 7.46169123980563, 0.963300196876966,
+                      list(60, 48.2744318767126, 49.8666666666667, 7.46169123980563, 0.812379616418173,
```

Comment:
So this is the result of the change to credible interval calculations?

Reply:
No, this is the result from removing columns for individual variables and showing only the difference between the pairs.